service/handlers_app_user.go

```diff
@@ -111,6 +114,14 @@ func SystemUserAssertionHandler(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	// Fetch the account assertions from the store
+	accountAssertions, err := fetchAccountAssertions(model)
+	if err != nil {
+		logMessage("USER", "account-assertions", err.Error())
+		formatBooleanResponse(false, "account-assertions", "", "Error retrieving the account assertions from the store", w)
```

Comment:
I've tried to be consistent with it. The admin UI is probably a bit more lenient as it is aimed at a limited number of sysadmins. The signing service and the system-user service are more "end-user" oriented, so I've tried to limit the raw messages that are sent back.